### PR TITLE
Enhance `drm` command, add `drm xmm0` functionality 

### PR DESCRIPTION
**Pull request by koyaan**
This is for the proposal in https://github.com/radareorg/radare2/issues/15213
Tests: https://github.com/radareorg/radare2-regressions/pull/1977

### CODE REVIEW

**Comment by radare** [comment]
i was thinking in supporting this in the r2 way.. so using a subcommand of drm instead of a special syntax for the arg. like: drmw drmh, drmq.. what do you think?

thanks for the contrib!

**Comment by koyaan** [comment]
That is a good idea, i will try to implement that! Is there a way to alias for example `dmmb xmm0` to `dmm xmm0 * 8` or should I remove the wildcard functionality?

**Comment by radare** [comment]
you can call the same command again with the wildcard syntax from the subcommand one if you like this way it will support both syntaxes.. is this a gdb thing ?

**Comment by koyaan** [comment]
No I was just looking for way to have index unspecified while specifying a size without breaking existing functionality.

**Comment by radare** [comment]
then i would go for just supporting the r2 subcommand way which is more consistent with the rest of r2

**Comment by koyaan** [comment]
I added the shorthand versions now, also updated tests.

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (radare) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb]: Wat is dis

File: libr/debug/p/native/linux/reg/linux-x64.h [hunk 2]
```diff
@@ -144,28 +144,36 @@ return strdup (
 "fpu	st6	.64	128	0\n"
 "fpu	st7	.64	144	0\n"
 
-"fpu	xmm0h	.64	160	0\n"
-"fpu	xmm0l	.64	168	0\n"
+"fpu    xmm0    .64 160 16\n"
```
Comment (radare) [inline libr/debug/p/native/linux/reg/linux-x64.h]: Can u update the other x64 profiles. Too? Not just the linux one

File: kyn-vectors.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+```
+20:27 <@r2tg> <Maijin> koyaan: x86.nz (x86 handmade assembler) is one of the many x86 assemblers
```
Comment (radare) [inline kyn-vectors.md]: Wtf

File: libr/core/cmd_debug.c [hunk 4]
```diff
@@ -2531,44 +2543,101 @@ static void cmd_debug_reg(RCore *core, const char *str) {
...
+								for (i = 0; i < item->packed_size / pack_sizes[pi]; i++) {
+									ut64 res = r_reg_get_pack(core->dbg->reg, item, i, pack_sizes[pi]);
+									pack_print(i, res, pi);
+								}
+								r_cons_printf("\n");
+							}
+						}
+					} else {
+						// print selected index / wordsize
+						r_cons_printf("0x%08"PFMT64x"\n", res);
```
Comment (radare) [inline libr/core/cmd_debug.c]: Missing space before (

File: libr/core/cmd_debug.c [hunk 5]
```diff
@@ -2531,44 +2543,101 @@ static void cmd_debug_reg(RCore *core, const char *str) {
...
-			if (p) {
-				*p++ = 0;
-				q = strchr (p, ' ');
-				if (q) {
-					*q++ = 0;
-					size = r_num_math (core->num, q);
+			char *q, *p, *name;
+			char *eq = NULL;
+			if(str[1] == ' ') {
+				name = strdup (str+2);
```
Comment (radare) [inline libr/core/cmd_debug.c]: Missing space around +

**Comment by koyaan** [comment]
Removed accidental files, and adjusted C-style, also added [?] indicator to help

File: libr/debug/p/native/linux/reg/linux-x64.h (hunk 2 above)
Comment (koyaan) [reply libr/debug/p/native/linux/reg/linux-x64.h]: Which ones? Windows does its own thing for example

**Comment by koyaan** [comment]
now mmx registers also work as expected
```
[0x004000b0]> drm mm0
0x00 0x70 0xa0 0x8a 0x36 0xfd 0xec 0x91
0x7000 0x8aa0 0xfd36 0x91ec
0x8aa07000 0x91ecfd36
0x91ecfd368aa07000
```

**Comment by radare** [comment]
The changes proposed looks good to me. Thanks!!

**Pull request merged**